kernel/src/main/scala/cats/kernel/std/list.scala

```diff
@@ -0,0 +1,95 @@
+package cats.kernel
+package std
+
+import scala.annotation.tailrec
+import scala.collection.mutable
+
+package object list extends ListInstances
+
+trait ListInstances extends ListInstances1 {
+  implicit def listOrder[A: Order] = new ListOrder[A]
+  implicit def listMonoid[A] = new ListMonoid[A]
+}
+
+trait ListInstances1 extends ListInstances2 {
```

Comment:
It's always easier to remove `sealed` and `private` than it is to add them in later.